editor/src/utils/utils.ts

```diff
@@ -900,7 +900,7 @@ function defer<T>(): Promise<T> & {
   return promise as any
 }
 
-export function keepReferenceIfDeepEqualSLOW<T>(original: T, maybeNew: T, measure = false) {
+export function keepReferenceIfDeepEqualSLOW<T>(original: T, maybeNew: T, measure = false): T {
   if (measure) {
     performance.mark('before eq')
   }
```

Comment:
the R.equals from here should be changed to fast-deep-equals

Reply:
I did do this at one point in a previous, but reverted it as I didn't actually want the change connected to it.